lib/checks/canDistribute.js

```diff
@@ -0,0 +1,24 @@
+const Node = require('../node');
+
+function canDistribute(node) {
```

Comment:
Not sure if this is the best name for this function/file. I'd be in favor of something more descriptive/accurate even if it's long, like `isBinomialSquared` or something? Wondering what Evy thinks.